introduction.md

```diff
@@ -0,0 +1 @@
+Hi I am the orc who is very hungry. And ugly.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
Done.